partials/forms/input-text-group.html

```diff
@@ -1,4 +1,4 @@
-<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{^date}}{{#error}} error{{/error}}{{/date}}">
+<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{^date}}{{#error}} error{{/error}}{{/date}} " {{#attributes}} {{attribute}}="{{value}}" {{/attributes}}>
```

Comment:
Custom attributes are added directly to text and textarea inputs. I think it's only when a field has options that we need to add attributes to a group. Although, thinking about it a little more, I wonder if using the same option (in this case attributes) in different places confuses things. Maybe `groupAttributes` would be better. What do other people think?